zerver/worker/queue_processors.py

```diff
@@ -2,7 +2,11 @@
 from __future__ import absolute_import
 from typing import Any, Callable, Dict, List, Mapping, Optional, cast
 
-from contrib_bots.bot_lib import BotHandlerApi, StateHandler
+import sys
+import os
+sys.path.insert(0, os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '../../api')))
+
+from bots_api.bot_lib import BotHandlerApi, StateHandler
```

Comment:
Couldn't directly import from `api.bots_api.bot_lib` but adding specifically the `api` folder to path allowed importing `bots_api.bot_lib`. No clue why.